jupyter/testing/ubi9-python-3.12/start-notebook.sh

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Load bash libraries
+SCRIPT_DIR=/opt/app-root/bin
+source ${SCRIPT_DIR}/utils/process.sh
```

Shellcheck (0.11.0)

[info] Not following: ./utils/process.sh was not specified as input (see shellcheck -x).

(SC1091)